apps/odo/test/integration/odo_integration_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Odo.IntegrationTest do
+  use ExUnit.Case
+
+  test "does nothing" do
+    # This test is to allow the umbrella CI script that runs
+    # integration tests across all sub-applications to run in the
+    # Odo app's context without re-running unit tests (and breaking)
```

Comment:
Thank you for taking the time to write this comment.

Reply:
We didn't leave our initials and a date, but hopefully you'll forgive the omission